src/Windows/AzureDCAPSetupProject/Product.wxs

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
-  <Product Id="*" Name="Azure DCAP Client" Language="1033" Version="1.0.0.0" Manufacturer="Microsoft" UpgradeCode="8c473d43-cce9-448e-b3ad-3858f4fe6459">
+  <Product Id="*" Name="AzureDCAPClient" Language="1033" Version="1.0.0.0" Manufacturer="Microsoft" UpgradeCode="8c473d43-cce9-448e-b3ad-3858f4fe6459">
```

Comment:
Should probably call it MicrosoftAzureDCAPClient based on feedback from Azure SDK folks.

Reply:
Fixed